app/views/abilities/show.html.erb

```diff
@@ -0,0 +1,110 @@
+                    </div>
+                <% else %>
+                    <p>You need to have many well-received posts.</p>
+                    <div class="meter is-progress">
+                        <div class="meter--bar is-100%"></div>
+                        <div class="meter--label">100%</div>
+                    </div>
+                <% end %>
+            <% end %>
+
+            <% unless @ability.edit_score_threshold.nil? %>
+                <% edit_score_percent = (linearize_progress(current_user.community_user.edit_score) / linearize_progress(@ability.edit_score_threshold) * 100).to_i %>
+
+                <p><strong>Edit score threshold</strong></p>
+                <% if edit_score_percent < 100 %>
+                    <p>You need to have more accepted suggested edits.</p>
+                    <div class="meter is-progress">
+                        <div class="meter--bar is-<%= edit_score_percent %>%"></div>
+                        <div class="meter--label"><%= edit_score_percent %>%</div>
+                    </div>
+                <% else %>
+                    <p>You need to have many accepted suggested edits.</p>
+                    <div class="meter is-progress">
+                        <div class="meter--bar is-100%"></div>
+                        <div class="meter--label">100%</div>
+                    </div>
+                <% end %>
+            <% end %>
+
+            <% unless @ability.flag_score_threshold.nil? %>
+                <% flag_score_percent = (linearize_progress(current_user.community_user.flag_score) / linearize_progress(@ability.flag_score_threshold) * 100).to_i %>
+
+                <p><strong>Flag score threshold</strong></p>
+                <% if flag_score_percent < 100 %>
+                    <p>You need to have more helpful flags.</p>
+                    <div class="meter is-progress">
+                        <div class="meter--bar is-<%= flag_score_percent %>%"></div>
+                        <div class="meter--label"><%= flag_score_percent %>%</div>
+                    </div>
+                <% else %>
+                    <p>You need to have many helpful flags.</p>
+                    <div class="meter is-progress">
+                        <div class="meter--bar is-100%"></div>
+                        <div class="meter--label">100%</div>
+                    </div>
+                <% end %>
+            <% end %>
+        </div>
+        <% end %>
+    <% end%>
+</div>
+<% else %>
+<div class="widget">
+    <div class="widget--body">
+        <p class="h-m-0"><i class="fas fa-fw fa-check h-c-green-600"></i> You have already earned this ability.</p>
+    </div>
+    <% if @your_ability&.is_suspended %>
+    <div class="widget--body">
+        <div class="notice is-danger">
+            <% if @your_ability.suspension_end.nil? %>
+                <p><strong>Your use of the <%= @ability.name %> ability has been suspended.</strong></p>

```

Comment:
We're only showing these to the user, right?

Reply:
The ability pages can be seen for any user by any user, but the suspension information is only shown to moderators and the user.